hash/msx1_cart.xml

```diff
@@ -12507,12 +12400,11 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
-<!-- Softcard -->
 	<software name="wreck">
 		<description>The Wreck (Euro)</description>
```

Comment:
Did you want to fully remove this one from msx1_cart.xml?

Reply:
No, the game was released on soft card and regular cartridge.

Reply:
Oh sorry, I see. Now the serial change etc. make sense.